examples/node/allowance-ERC20.js

```diff
@@ -0,0 +1,26 @@
+const Matic = require('maticjs').default
+const config = require('./config')
+
+const token = config.ROPSTEN_TEST_TOKEN // test token address
```

Comment:
`ROPSTEN_TEST_TOKEN` should be `PARENT_TEST_TOKEN` why keep specific network name in the codebase.

Reply:
@arthcp can you take this up?

Reply:
Sure! I'll make the changes.